src/components/BooksList.js

```diff
@@ -0,0 +1,28 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import SingleBook from './SingleBook';
+import { gettingBooks } from '../redux/books/booksActions';
+
+function BooksList() {
+  const books = useSelector((state) => state.booksReducer);
+  const dispatch = useDispatch();
+
+  useEffect(() => { dispatch(gettingBooks()); }, []);
+
+  return (
+    <ul>
+      {books.map(
+        (book) => (
+          <SingleBook
+            key={book.item_id}
+            item_id={book.item_id}
+            category={book.category}
+            title={book.title}
+          />
+        ),
+      )}
```

Comment:
✅ BookList is required to be able to show a list of books using a book component to display each individual book 👍🏻